stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/BaseMessageItemViewHolder.kt

```diff
@@ -3,27 +3,22 @@ package io.getstream.chat.android.ui.messages.adapter
 import android.view.View
 import androidx.recyclerview.widget.RecyclerView
 import com.getstream.sdk.chat.adapter.MessageListItem
-import io.getstream.chat.android.ui.messages.adapter.viewholder.decorator.Decorator
 
 public abstract class BaseMessageItemViewHolder<T : MessageListItem>(
     itemView: View,
-    private val decorators: List<Decorator>
```

Comment:
Decorator related functionality was moved to another, `internal` base class, `DecoratedBaseMessageItemViewHolder`